pkg/actions/lua/lakefs/catalogexport/delta_exporter.lua

```diff
@@ -34,6 +35,22 @@ local function delta_log_entry_key_generator()
     end
 end
 
+
+-- Local function to get the table descriptor
+local function get_table_descriptor(repo, ref, table_name_yaml, table_descriptors_path)
+    local tny = table_name_yaml
+    if not strings.has_suffix(tny, ".yaml") then
+        tny = tny .. ".yaml"
+    end
+    local table_src_path = pathlib.join("/", table_descriptors_path, tny)
+    local table_descriptor = extractor.get_table_descriptor(lakefs, repo, ref, table_src_path)
+    return table_descriptor
+end
+
+local function starts_with(str, start)
+    return string.sub(str, 1, #start) == start
+end
```

Comment:
You already have a `strings.hasPrefix` [function](https://github.com/treeverse/lakeFS/blob/master/pkg/actions/lua/strings/strings.go#L53C6-L53C15) implemented in the strings package